Tests/Unit/Form/Type/PathTypeTest.php

```diff
@@ -72,12 +71,4 @@ public function testBuildForm()
 
         $this->type->buildForm($this->builder, array('manager_name' => null));
     }
-
-    public function testSetDefaultOptions()
```

Comment:
didn't really see the value in this test and keeping it alive is just complicated due to the API changes in the form layer